### PR TITLE
Reboot to BIOS

### PR DESCRIPTION
**Pull request by benadida**
As part of #1862, add an option to reboot into BIOS on the superadmin screen.

### CODE REVIEW

File: libs/test-utils/src/fake_kiosk.ts [hunk 1]
```diff
@@ -99,6 +99,8 @@ export function fakeKiosk({
     sign: jest.fn(),
     log: jest.fn(),
     reboot: jest.fn(),
+    // eslint-disable-next-line vx/gts-identifiers
+    rebootToBIOS: jest.fn(),
```
Comment (jonahkagan) [inline libs/test-utils/src/fake_kiosk.ts:103]: Nit: I know that it's weird to follow the camel case convention for acronyms, but the rest of the code does it (e.g. see `Usb` below and `Ipp` elsewhere), so I don't know that there's a strong reason to make an exception here.

File: frontends/bmd/src/pages/superadmin_screen.test.tsx [hunk 2]
```diff
@@ -20,14 +20,26 @@ it('the right buttons and reset calls quit', () => {
...
 
   // test without kiosk
   fireEvent.click(screen.getByText('Reset'));
 
   // test with kiosk
   window.kiosk = fakeKiosk();
-  fireEvent.click(screen.getByText('Reset'));
-
-  expect(window.kiosk.quit).toHaveBeenCalledTimes(1);
+  act(() => {
```
Comment (jonahkagan) [inline frontends/bmd/src/pages/superadmin_screen.test.tsx:31]: I think `act` is unnecessary here. Was there a specific reason for adding it?

**Review by jonahkagan — APPROVED** [review]

File: libs/test-utils/src/fake_kiosk.ts (hunk 1 above)
Comment (benadida) [reply libs/test-utils/src/fake_kiosk.ts:103]: grumble... you are right, even as I find this particular decision annoying :)

File: frontends/bmd/src/pages/superadmin_screen.test.tsx (hunk 2 above)
Comment (benadida) [reply frontends/bmd/src/pages/superadmin_screen.test.tsx:31]: I'm pretty sure there was a warning until I added it, but I'll check.

Comment (benadida) [reply frontends/bmd/src/pages/superadmin_screen.test.tsx:31]: huh, I was convinced it asked for it, but turns out it didn't, you are right. Removing.

**Pull request merged**